packages/web3-providers/src/opensea/index.ts

```diff
@@ -107,7 +107,7 @@ function createNFTToken(chainId: ChainId, asset: OpenSeaResponse): NonFungibleTo
             address: asset.token_address ?? asset.asset_contract.address,
             name: asset.name ?? asset.collection.name,
             symbol: asset.asset_contract.symbol,
-            owner: asset.owner.address,
+            owner: asset.top_ownerships[0].owner.address || asset.owner.address,
```

Comment:
Per the [doc](https://docs.opensea.io/reference/retrieving-a-single-asset), they mean different things.

![image](https://user-images.githubusercontent.com/52657989/184542809-b1fded3a-84a4-4e90-9932-1bd809e1c980.png)